doc/using-php-5.3.rst

```diff
@@ -0,0 +1,20 @@
+Using the SDK with PHP v5.3
```

Comment:
Great idea to create this document. Do you think we should link to it from http://docs.php-opencloud.com/en/latest/#installation?